clients/cli/src/main.rs

```diff
@@ -214,6 +216,35 @@ enum Lockbook {
 
     /// Subcommands that aid in extending lockbook
     Debug(Debug),
+
+    /// Subcommands for admin users
+    #[structopt(setting(AppSettings::Hidden))]
+    Admin(Admin),
+}
+
+#[derive(Debug, PartialEq, StructOpt)]
+pub enum Admin {
+    /// Commands related to feature flags
+    FeatureFlags(FeatureFlags),
+
+    /// Delete a user
+    DeleteAccount { username: String },
+}
+
+#[derive(Debug, PartialEq, StructOpt)]
+pub enum FeatureFlags {
+    /// Prints out the current state of all feature flags
+    List,
+
+    /// Subcommands of the new account feature flag
+    NewAccounts(FeaturesSwitchOptions),
+}
+
+#[derive(Debug, PartialEq, StructOpt)]
+pub enum FeaturesSwitchOptions {
+    ToggleOn,
+
+    ToggleOff,
```

Comment:
friends don't let friends re-invent booleans

Reply:
I didn't do the boolean stuff like we had in the old admin cli since it felt wrong. Unless you have something else in mind?